### ISSUE TITLE
Broken passing of arguments in plot.model_parts and plot.predict_parts.

### ISSUE THREAD

**Issue by pbiecek** [ModelOriented/DALEX#277]
critical:
new plot functions silently ignore all arguments passed to plot.model_parts and plot.predict_parts.
```
bd_rf_distr <- predict_parts(explainer = explain_rf,
                       new_observation = henry, 
                                  type = "break_down",
              order = c("age", "class", "fare", "gender", 
                        "embarked", "sibsp", "parch"),
                    keep_distributions = TRUE)
plot(bd_rf_distr, plot_distributions = TRUE, max_features = 3) 
```

**Comment by pbiecek** [comment]
candidate fix in 5e88297873b5e9e029d45d0ec45c8289a86abae6

**Comment by pbiecek** [comment]
tested in EMA
